### DISCUSSION TITLE
Vector stores errors - Postgres and Vectara

### DISCUSSION THREAD

**Discussion by datasci888** [FlowiseAI/Flowise#1906] [Q&A]
Hi 
When I add Postgres and add credentials, host, and database,  error:connection is insecure (try using sslmode=require) is shown. I set up the PostgreSQL server using jupyter notebook, created the database, and verified embeddings. The database accepts connections, and I tested it with an SQL query. The database was accessed through neondb. 

Similarly, the Vectara node was giving an upsert error when I used the pdf document loader. 
Error: Error: Error Vectara API returned status code FORBIDDEN: undefined while adding document

We have checked the credentials and all seems good. Any guidance would be highly appreciated.

**Comment by Jaredude** [comment]
To use a Postgres DB that requires sll (neon requires sll), you'll need to set an ENV variable of 
`DATABASE_SSL=true`
of
`PGSSLMODE=require`

**Comment by ml4den** [comment]
`Error: vectorsService.upsertVector - Error: error: connection is insecure (try using `sslmode=require`)`

`PGSSLMODE=require` - Use this for connections to Postgres databases inside of Flowise, such as Record Manager nodes, and to fix the error above. This is good practice and required for some DBs like Neon.

`DATABASE_SSL=true` - Also good practice, use this for your Flowise DB itself.

**Reply by ferchobanana** [reply]
I have the same issue and i set both ENV variables, PGSSLMODE=require and DATABASE_SSL=true and still get exactly the same problem "error:connection is insecure (try using sslmode=require)" any suggestion? :c
